jmix-flowui/flowui/src/main/java/io/jmix/flowui/view/ViewControllerUtils.java

```diff
@@ -136,4 +136,8 @@ public static <T> Registration addInitEntityEvent(StandardDetailView<T> view, Co
     public static Registration addBeforeShowEvent(View<?> view, ComponentEventListener<View.BeforeShowEvent> listener) {
```

Comment:
but idk why add-listener named "addBeforeShow**Event**", but i think `addValidationEventListener` is better then `addValidationEvent`

Reply:
It seems that that you should ask yourself about this naming:)

![image](https://github.com/jmix-framework/jmix/assets/7888486/85c77a93-5864-4c01-a4a9-5650482e793f)

Name with the Listener suffix seems more correct, so analyze if it's possible to rename all methods that you added previously.

Reply:
Fixed in: https://github.com/jmix-framework/jmix/issues/2885